src/main.js

```diff
@@ -0,0 +1,271 @@
+                    <p>This could be because:</p>
+                    <ul class="govuk-list govuk-list--bullet">
+                        <li>the paper LPA hasn’t been scanned yet</li>
+                        <li>the name is spelt incorrectly or differently on the paper LPA which means a match can’t be found</li>
+                    </ul>
+
+                    <p>For the best search results try searching with the case number.</p>
+
+                    <p>If you don’t have the case number try searching by:</p>
+                    <ul class="govuk-list govuk-list--bullet">
+                        <li>full name and date of birth, or</li>
+                        <li>full name and postcode</li>
+                    </ul>
+                </div>
+            `);
+    }
+    return;
+  }
+
+  this.render(`
+        <ul class="${CLASSES.list}">
+        ${results
+          .map(
+            (result) => `
+                <li class="${CLASSES.item}">
+                    <strong>
+                      ${escapeHTML(result.firstname)} ${escapeHTML(
+                        result.surname
+                      )} [${result.personType}]
+                    </strong>
+                    <p class="${CLASSES.link}">
+                        <a class="govuk-link" href="/lpa/person/${result.id}/${
+              result.case.id
+            }">
+                            ${result.case.uId}
+                            ${
+                              result.case.caseType
+                            }/${result.case.caseSubtype.toUpperCase()}
+                        </a>
+                    </p>
+                    <dl class="govuk-summary-list govuk-summary-list--no-border govuk-!-margin-bottom-0">
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key govuk-!-padding-1"><abbr title="Date of birth">DOB:</abbr></dt>
+                            <dd class="govuk-summary-list__value govuk-!-padding-1">${
+                              result.dob
+                            }</dd>
+                        </div>
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key govuk-!-padding-1">Address:</dt>
+                            <dd class="govuk-summary-list__value govuk-!-padding-1">${
+                              result.addresses
+                                ? formatAddress(result.addresses[0])
+                                : ""
+                            }</dd>
+                        </div>
+                        ${
+                          result.case.status
+                            ? `
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key govuk-!-padding-1">Status:</dt>
+                            <dd class="govuk-summary-list__value govuk-!-padding-1">${formatStatus(
```

Comment:
could probably inline the HTML from this here and just call a function to get the correct class colour